unit/GetBaseActorValuesTest.cpp

```diff
@@ -0,0 +1,32 @@
+#include "TestUtils.hpp"
+#include <GroupUtils.h>
+#include <Loader.h>
+#include <catch2/catch.hpp>
+
+#include "GetBaseActorValues.h"
+
+extern espm::Loader l;
+PartOne& GetPartOne();
+
+TEST_CASE("GetBaseActorValues works correctly", "[GetBaseActorValues]")
+{
+  PartOne& p = GetPartOne();
+  DoConnect(p, 0);
+  p.CreateActor(
+    0x0001B1DB, { 0, 0, 0 }, 0,
+    0x3c); // Ri'saad's id. (Roving merchant from Khajiit's caravan.)
```

Comment:
Ri'saad's id is probably about `0x0001B1DB`. Also this looks more clean in my opinion (with number arguments stated explicitly), but I think I have no right to enforce it :)
```c++
  p.CreateActor(
    /*formId*/ 0x0001B1DB, // Ri'saad's id. (Roving merchant from Khajiit's
                           // caravan.)
    /*pos*/ { 0, 0, 0 }, /*cellOrWorld*/ 0,
    /*profileId*/ 0x3c);
```

Reply:
Hmm. BTW `0x0001B1DB` occurs here quite frequently. What about making a constant above? In current code style this would look like this:
```c++
// Ri'saad. Roving merchant from Khajiit's caravan
const uint32_t g_risaadFormId = 0x0001B1DB;
```